src/components/m-datepicker-body/_template.js

```diff
@@ -0,0 +1,14 @@
+import html from 'nanohtml';
+
+const renderCells = (store) => {
+  if (!store) {
+    return [];
+  }
+  return store.getCells();
```

Comment:
shouldn't that be done outside of template?

Reply:
where? cause the FB u gave last time was to do it inside the template for DOM diffing and the cells are filled from the "model" inside datepicker-body.js. 

Open for suggestions :)

Reply:
hmh, I thought that the `store` will provide those cells by default.

Reply:
they have to be calculated related from date object and those cells need to be used in the model and in the template